modules/update-service-overview.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='update-service-overview-{context}']
+== The {product-title} update service
+
+The {product-title} update service is the hosted service that provides over the air updates to both 
+{product-title} and RHCOS. It provides a graph, or diagram that contain
+_vertices_ and the _edges_ that connect them, of component Operators. The edges
+in the graph show which versions you can safely upgrade to. The cluster version
```

Comment:
Before you talk about edges for upgrades, you should probably point out that the vertices are update payloads that specify the intended state of the managed cluster components.